apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SqlResultStream.cs

```diff
@@ -111,7 +111,12 @@ public async Task<bool> MoveNext(CancellationToken cancellationToken)
         private async Task<PartialResultSet> ComputeNextAsync(CancellationToken cancellationToken)
         {
             // The retry state is local to the method as we're not trying to handle callers retrying.
-            RetryState retryState = new RetryState(_client.Settings.Scheduler ?? SystemScheduler.Instance, _retrySettings);
+            // Consecutive retryable errors will be retried until the call succeeds, or until the call times out or is cancelled.
+            RetryState retryState = new RetryState(_client.Settings.Scheduler ?? SystemScheduler.Instance, _retrySettings, maxConsecutiveErrors: int.MaxValue);
+            var clock = _client.Settings.Clock ?? SystemClock.Instance;
+            var startTime = clock.GetCurrentDateTimeUtc();
+            var originalTimeout = _callSettings.Expiration?.Timeout ?? _client.Settings.ExecuteStreamingSqlSettings.Expiration?.Timeout ?? TimeSpan.FromMilliseconds(3600000);
```

Comment:
nit: can the ultimate default value (the 1hr) be set somewhere in a  static variable or similar?